decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,95 @@
+---
+status: accepted 
+date: 2023-12-19
+deciders: Chris Mungall, Lee Ann 
+consulted: Anastasiya Prymolenna, Brynn Zalmanek, James Tessmer, Montana Smith, Sam Purvine, Yuri Corilo, Mark Miller, Michael Thorton, Alicia Clum, Mark Miller    
+informed: Emiley Eloe-Fadrosh, Shreyas Cholia, Eric Cavanna 
+---
+# Schema Refactoring: Monterey//Berkeley
+
+## Decisions Made
+
+### Class:PlannedProcess
+* `Class:PlannedProcess` will be created as a root class 
+* All "Activity" classes are now of `Class:PlannedProcess`, and the word "Activity" will be removed
+
+### Class:MaterialProcessing
+* `Class:MaterialProcessing` will be created and be a subclass of `Class:PlannedProcess`
+* All processes involving material to material transformations (Biosamples to ProcessedSample, ProcessedSample to ProcessedSample) will be subclasses of `Class:MaterialProcessing` (examples: `Class:Extraction`, `Pooling`, `MixingProcess`, etc.). 
+* The output of any subclasses of `Class:MaterialProcessing` will have an output of ProcessedSample
+* `Class:ChemicalConversionProcess` will replace `Class:ReactionProcess`
+* `Class:BiosampleProcessing` will be removed. Subclasses of `Class:MaterialProcessing` will replace what was captured. 
```

Comment:
Some of the bullets end with a period (`.`) and some don't. I suggest ending them all with one (assuming they are all sentences).